### DISCUSSION TITLE
How to get my rtl_433.conf into a container

### DISCUSSION THREAD

**Discussion by HolgerAusB** [hertzg/rtl_433_docker#108] [Q&A] [answered]
I am using rtl_433 for some years now by using the 'building' option. But I am not very good in coding and building things. So I never updated this installation. But it works.

Now I want to use a new Raspberry Pi 4 for all of my Smart Home things and also move the rtl-stick to this. I want to give the docker version a try and using the docker-compose.yml option. But I want to reuse my rtl_433.conf which contains two flex devices (-X) for my doorbells.

Would it be enough to set the following to the `docker-compose.yml` file?:
```yml
volumes:
      - ./data:/etc/rtl_433
```
and removing the command section?

**Comment by hertzg** [comment, marked as answer]
Looking at the `-h` output of the process you will see the following message that can be a good hint to what to try

```
A "rtl_433.conf" file is searched in "./", XDG_CONFIG_HOME e.g. "$HOME/.config/rtl_433/",
  SYSCONFDIR e.g. "/usr/local/etc/rtl_433/", then command line args will be parsed in order.
```

There's also `[-c <path>] Read config options from a file` if all previous fails you can put the file anywhere in the container (volume mount would do fine) and use the `-c /path/to/that/file.conf` if all else fails.
